distributed/s3.py

```diff
@@ -2,16 +2,22 @@
 
 import logging
 import threading
+import re
+import json
+import io
+import sys
 
 import boto3
-from botocore.handlers import disable_signing
+from botocore.exceptions import ClientError
 from tornado import gen
 
 from dask.imperative import Value
+from dask.base import tokenize
+from distributed.utils import read_block, seek_delimiter
```

Comment:
Use relative imports within source files.  See the `.utils` import you've added a few lines below.